src/libpriv/rpmostree-importer.cxx

```diff
@@ -615,11 +615,13 @@ append_tmpfiles_d (RpmOstreeImporter *self,
  * https://github.com/projectatomic/rpm-ostree/issues/233
  */
 static gboolean
-path_is_ostree_compliant (const char *path)
+path_is_ostree_compliant (const char * const path)
 {
+  g_assert (path != NULL);
   g_assert (*path == '/');
-  path++;
```

Comment:
"mutating" is a strong/misleading word here - this function is *not* changing any heap memory.  It's just incrementing the stack-allocated pointer value.

We do *dereference* via `*path` but that's just reading.  Actually *mutating* would be more like `*path = '\0'` or so.

`const char * const` is indeed more correct but the problem here is that `const` is annoying in C except for a few cases because of how inconsistently it is used in library APIs.  

This patch shouldn't hurt anything, and I'm ok to merge if you want, but I don't think it fixes anything either.
(We could probably oxidize it instead)